### PR TITLE
Docker compose checks to ensure that paths are mounted correctly and files are there

### PR DESCRIPTION
**Pull request by nolim1t**
Docker compose checks to ensure that paths are mounted correctly and files are there. just to ensure robustness.

Will also put this into #185 as well

### CODE REVIEW

File: etc/init.d/docker-compose [hunk 1]
```diff
@@ -8,10 +8,28 @@ depend() {
...
 start() {
   ebegin "Starting docker-compose"
-
-  cd /home/lncm/compose
-  docker-compose up -d
-
+  if [[ -d /media/important/important/lnd ]]; then
+    if [[ -d /media/archive/archive/bitcoin ]]; then
+      if [[ -f /media/important/important/lnd/lnd.conf ]]; then
+        if [[ -f /media/archive/archive/bitcoin/bitcoin.conf ]]; then
```
Comment (AnotherDroog) [inline etc/init.d/docker-compose]: the `if` statements would be easier to deal with when they are not nested

Comment (nolim1t) [reply etc/init.d/docker-compose]: what linting errors

Comment (nolim1t) [reply etc/init.d/docker-compose]: I just pushed || exit to the lines

**Comment by nolim1t** [comment]
let me know if this is ok @AnotherDroog

**Comment by AnotherDroog** [comment]
No, have you used shellcheck?

**Comment by nolim1t** [comment]
Yes

**Comment by AnotherDroog** [comment]
I cleaned it up

Note, I changed `mana` to `noma`, which isn't merged in yet - although there is no reason not to merge that PR in too

**Pull request merged**